services/src/messaging/src/main/java/org/openkilda/messaging/nbtopology/request/FeatureTogglesBaseRequest.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.nbtopology.request;
+
+@SuppressWarnings("squid:MaximumInheritanceDepth")
+public class FeatureTogglesBaseRequest extends BaseRequest {
```

Comment:
For my point, it's better to remove this intermediate class

Reply:
We can not delete this class. This class uses to identify any feature toggles request in Nbworker.